.github/workflows/ci.yml

```diff
@@ -272,6 +272,18 @@ jobs:
       run: |
         cd src/ApiService/
         dotnet test --no-restore
+    - name: copy artifacts
+      run: |
+        cd src/ApiService/ApiService/
+        echo ${GITHUB_RUN_ID} | tee onefuzzlib/build.id
+        echo ${GITHUB_SHA} | tee onefuzzlib/git.version
+        zip -r api-service-net.zip bin/Release/*
+        mkdir -p ${GITHUB_WORKSPACE}/artifacts/service-net
+        cp api-service-net.zip ${GITHUB_WORKSPACE}/artifacts/service-net
```

Comment:
we need to generate the version number. 
You can see call getversion.sh to put the verision number in a file.
We also need to publish that version number during the 
deployment

Reply:
I've setup generating the version number and adding the local.settings.json file we need in the deployment in another branch from my [own fork](https://github.com/AdamL-Microsoft/onefuzz/blob/e2f99dbe2d06c55e97646ffd64be58c00128f1f0/.github/workflows/ci.yml#L277)

I'll close this PR out in favor of using the changes from the new PR I'll create from that fork I've worked on, and link it back to the parent issue #1752